plugins/woocommerce/includes/admin/views/html-admin-settings.php

```diff
@@ -20,7 +20,8 @@
 }
 
 $hide_nav = Features::is_enabled( 'reactify-classic-payments-settings' ) &&
-	'checkout' === $current_tab && 'offline' === $current_section;
+	( ( 'checkout' === $current_tab && 'offline' === $current_section ) ||
+	( 'checkout' === $current_tab && 'recommended' === $current_section ) );
```

Comment:
Nit: Should we simplify this?
```suggestion
	( 'checkout' === $current_tab && ( 'offline' === $current_section || 'recommended' === $current_section ) );
```

Reply:
Agreed, the current conditional is quite hard to read.